staging/src/k8s.io/client-go/examples/leader-election/main.go

```diff
@@ -124,38 +121,29 @@ func main() {
 		// get elected before your background loop finished, violating
 		// the stated goal of the lease.
 		ReleaseOnCancel: true,
-		LeaseDuration:   60 * time.Second,
-		RenewDeadline:   15 * time.Second,
-		RetryPeriod:     5 * time.Second,
+		LeaseDuration:   15 * time.Second,
+		RenewDeadline:   10 * time.Second,
+		RetryPeriod:     2 * time.Second,
 		Callbacks: leaderelection.LeaderCallbacks{
 			OnStartedLeading: func(ctx context.Context) {
 				// we're notified when we start - this is where you would
 				// usually put your code
-				klog.Infof("%s: leading", id)
+				run(ctx)
 			},
 			OnStoppedLeading: func() {
-				// we can do cleanup here, or after the RunOrDie method
-				// returns
-				klog.Infof("%s: lost", id)
+				// we can do cleanup here
+				klog.Fatalf("leader lost: %s", id)
 			},
 			OnNewLeader: func(identity string) {
 				// we're notified when new leader elected
 				if identity == id {
 					// I just got the lock
 					return
 				}
-				klog.Infof("new leader elected: %v", identity)
+				klog.Infof("new leader elected: %s", identity)
 			},
 		},
 	})
 
-	// because the context is closed, the client should report errors
-	_, err = client.CoordinationV1().Leases(leaseLockNamespace).Get(leaseLockName, metav1.GetOptions{})
-	if err == nil || !strings.Contains(err.Error(), "the leader is shutting down") {
-		log.Fatalf("%s: expected to get an error when trying to make a client call: %v", id, err)
-	}
```

Comment:
what is the reason for removing that from the example?

Reply:
I think it's ilelegant as we need to confirm error with `strings.Contains()`,  and remove that can simplify the example.